uniflow/extract/config.py

```diff
@@ -9,10 +13,21 @@ class ExtractConfig:
 
     flow_name: str
     num_thread: int = 1
+    model_config: Optional[ModelConfig] = None
+    guided_prompt_template: Optional[GuidedPrompt] = None
```

Comment:
nit: same here. You should remove this because now flow requires GuidedPrompt for extract.